asset/mock.go

```diff
@@ -103,7 +104,15 @@ func RandAssetWithValues(t testing.TB, genesis Genesis, groupKey *GroupKey,
 		t.Fatal("unhandled asset type", genesis.Type)
 	}
 
-	a, err := New(genesis, units, 0, 0, scriptKey, groupKey)
+	var assetVersion Version
+	if prand.Int()%2 == 0 {
```

Comment:
Need to use `test.RandInt[uint8]()%2 == 0` to achieve stable/deterministic test vectors.